### PR TITLE
Separate VM conversion from pipeline logic

### PR DESCRIPTION
**Pull request by mikevanhemert**

### CODE REVIEW

**Review by docandrew — COMMENTED** [review]
Looks good to me - separating out the container build makes a lot of sense!

File: Dockerfile [hunk 1]
```diff
@@ -1,2 +1,45 @@
...
+  -i ova "disk.ova" \
+  -o local -of qcow2 -os "$PWD" -on "disk" && \
+  mv "disk-sda" "disk.qcow2" && \
+  virt-customize \
+  -a "disk.qcow2" \
+  --run-command 'yum install -y cloud-init qemu-guest-agent' \
+  --run-command 'cloud-init clean --logs --seed'  \
+  --append-line '/etc/default/grub:GRUB_TERMINAL=serial' \
+  --append-line '/etc/default/grub:GRUB_SERIAL_COMMAND="serial --speed=115200 --unit=0 --word=8 --parity=no --stop=1"' \
+  --run-command 'grub2-mkconfig -o /boot/grub2/grub.cfg'
```
Comment (bburky) [inline Dockerfile:42]: Not a serious issue, but some extra indentation makes this easier to read:
```suggestion
RUN \
  virt-v2v \
    -i ova "disk.ova" \
    -o local -of qcow2 -os "$PWD" -on "disk" && \
  mv "disk-sda" "disk.qcow2" && \
  virt-customize \
    -a "disk.qcow2" \
    --run-command 'yum install -y cloud-init qemu-guest-agent' \
    --run-command 'cloud-init clean --logs --seed'  \
    --append-line '/etc/default/grub:GRUB_TERMINAL=serial' \
    --append-line '/etc/default/grub:GRUB_SERIAL_COMMAND="serial --speed=115200 --unit=0 --word=8 --parity=no --stop=1"' \
    --run-command 'grub2-mkconfig -o /boot/grub2/grub.cfg'
```

**Review by bburky — APPROVED** [review]
LGTM

**Review by blancharda — APPROVED** [review]
image

Looks good to @bburky. 👍

**Pull request merged**